bindings/python/unicorn/unicorn.py

```diff
@@ -8,7 +8,7 @@
 
 __version__ = "%u.%u.%u" % (__MAJOR, __MINOR, __PATCH)
 
-if _sys.version_info.major == 2:
-    from .unicorn_py2 import *
-else:
+if _sys.version_info >= (3,):
```

Comment:
It could make sense if we were differentiating between different minor versions, but only comparing for major version this is less readable.